style.css

```diff
@@ -1,811 +1,836 @@
-  /* Works section */
-  #project-container {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-  }
-
-  .project-card {
-    display: flex;
-    flex-direction: row;
-    gap: 24px;
-    width: 80%;
-  }
-
-  .project-detail-contener {
-    padding: 24px;
-  }
-
-  .project-card img {
-    width: 50%;
-    height: 448px;
-    border-radius: 8px;
-  }
-
-  .project-card h2 {
-    font-family: "Poppins", sans-serif;
-    font-size: 40px;
-    font-weight: 700;
-    line-height: 52px;
-    color: 091e42;
-  }
-
-  .project-card h4 {
-    font-family: "Poppins", sans-serif;
-    font-size: 18px;
-    line-height: 24px;
-    font-weight: 500;
-    color: #6b778c;
-    padding: 8px 0;
-  }
-
-  .project-card h4 span {
-    color: #344563;
-  }
-
-  .project-card p {
-    font-family: "Poppins", sans-serif;
-    font-size: 16px;
-    font-weight: 400;
-    line-height: 24px;
-    color: #344563;
-  }
-
-  .tools {
-    font-family: "Poppins", sans-serif;
-    font-weight: 500;
-    font-size: 12px;
-    line-height: 16px;
-    color: #6070ff;
-  }
-
-  .see-project-btn {
-    font-family: "Poppins", sans-serif;
-    font-size: 17px;
-    font-weight: 500;
-    line-height: 24px;
-    text-align: center;
-    color: #396df2;
-    background-color: white;
-    border: 1px solid #6070ff;
-    padding: 12px;
-    border-radius: 8px;
-    margin-top: 16px;
-  }
-}
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Poppins", sans-serif;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.hamburger {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+}
+
+body {
+  position: relative;
+  background-color: #f7f7f9;
+  width: 100vw;
+  overflow-x: hidden;
+}
+
+.heading-large {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 52px;
+}
+
+header {
+  background-color: #fff;
+  color: #1a34ff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 25px;
+}
+
+.nav-links {
+  display: none;
+}
+
+.logo > a {
+  text-decoration: none;
+  font-weight: bold;
+  left: 76px;
+}
+
+/* start mobile-menu */
+.mobile-menu-close-icon {
+  color: #fff;
+  align-self: flex-end;
+  padding: 0;
+}
+
+.mobile-menu--hide {
+  display: none;
+  width: 0;
+  height: 0;
+}
+
+.mobile-menu--show {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 1em 0.75em;
+  width: 100vw;
+  height: 100vh;
+  color: #fff;
+  background: rgba(96, 112, 255, 0.9);
+  font-family: "Poppins", sans-serif;
+  font-weight: 600;
+  font-size: 2rem;
+  line-height: 44px;
+  backdrop-filter: blur(4px);
+}
```

Comment:
- [ ] Good job working on the mobile menu 🥇, however, if we try to scroll the page, we get some issues 😭, but let's just change some lines of code by giving our `.mobile-menu--show` a position `fixed`  and add a `z-index` propaty and hope that this one my help 😄. It can be something like this:
````css
.mobile-menu--show {
  ...
  position: fixed;
  z-index: 3;
  ...
}
```` 

![mob](https://github.com/patrick-amponsah/Portfolio-Project/assets/57408419/60abc1a8-b9c4-473d-8e28-9b5893a25697)

Kindly, fix it 👍